substrate/client/db/src/lib.rs

```diff
@@ -1486,6 +1486,7 @@ impl<Block: BlockT> Backend<Block> {
 				.map(|(n, _)| n)
 				.unwrap_or(Zero::zero());
 			let existing_header = number <= highest_leaf && self.blockchain.header(hash)?.is_some();
+			let body_exists = pending_block.body.is_some();
```

Comment:
Nit: to match `existing_header` naming above.
```suggestion
			let existing_body = pending_block.body.is_some();
```